src/MagnumPlugins/UfbxImporter/UfbxImporter.cpp

```diff
@@ -159,15 +166,55 @@ constexpr Containers::StringView sceneFieldNames[] = {
     "Visibility"_s,
 };
 
-struct MeshChunk {
-    uint32_t meshId;
-    uint32_t meshMaterialIndex;
-};
+Containers::StringView blendModeToString(ufbx_blend_mode mode) {
```

Comment:
I added it way up there so it's included in the anonymous `namespace {}`, should I introduce a mini-anonymous-namespace for it next to `doMaterial()` (I recall seeing that you're not a big fan of `static` in some style docs :D)

Reply:
Yep, anonymous namespaces are fine, any number of them anywhere in the file.

I'm not a fan of `static` because IIRC it doesn't work for class / struct definitions, those don't get the file-local linker treatment.